src/containers/language/language.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { Select, MenuItem } from '@material-ui/core';
+import {
+  setToLocalStorage,
+  getFromLocalStorage
+} from '../../services/localstorage.service';
+import useStyles from './language.styless';
+import { changeLanguage } from '../../redux/language/language.actions';
+import { LANGUAGES_LIST } from '../../configs';
+
+const languageInLocalStorage = JSON.parse(getFromLocalStorage('language')) || 0;
```

Comment:
you don`t need to parse it here. Localstorage service already did it for you

Reply:
fixed